spec/std/uri_spec.cr

```diff
@@ -322,6 +322,21 @@ describe "URI" do
     end
   end
 
+  describe "#query_params" do
+    it do
+      uri = URI.parse("http://foo.com?id=30&limit=5#time=1305298413")
+      uri.query_params.should eq(HTTP::Params.parse("id=30&limit=5"))
```

Comment:
Technically, the spec would be more reliant if the `HTTP::Params` instance was created from a spec. We're not testing `Params.parse` here, but still...

Reply:
What do you mean? Something like:

```crystal
expected_params = HTTP::Params.new({ "id" => ["30"], "limit" => ["5"]} )
uri.query_params.should eq(expected_params)
```